providers/chargebee/support.go

```diff
@@ -49,6 +64,17 @@ var objectNameWithListSuffix = datautils.NewSet( //nolint:gochecknoglobals
 	"currencies",
 )
 
+var objectNameWrite = datautils.NewDefaultMap(datautils.Map[string, string]{ //nolint:gochecknoglobals
```

Comment:
Please document the fact that we are converting these object names only because they are actions/verbs, and not resources